lib/src/signed_video_h26x_sign.c

```diff
@@ -109,6 +109,7 @@ static svi_rc
 complete_sei_nalu_and_add_to_prepend(signed_video_t *self)
 {
   assert(self);
+  if (self->payload_buffer_idx < 1) return SVI_NOT_SUPPORTED;
```

Comment:
Note that also `payload_buffer_idx == 1` is invalid, hence this should be `payload_buffer_idx < 2`